spec/features/js/creating_entities_from_list_members_spec.rb

```diff
@@ -0,0 +1,44 @@
+# Example feature spec to demonstrate JS interactions via Chromedriver
+feature 'Creating entities from list members page', type: :feature, js: true do
+  let(:list_owner) { create_basic_user }
+
+  before do
+    create(:person_extension_definition)
+    login_as list_owner, scope: :user
+  end
+
+  after do
+    logout(:user)
+  end
+
+  context 'with a list of people', :sphinx do
+    let(:list) { create(:list, name: 'The Crying of Lot 49', creator_user_id: list_owner.id, last_user_id: list_owner.id) }
+    let(:oedipa) { create(:entity_person, name: 'Oedipa Maas') }
+    let(:mucho) { create(:entity_person, name: 'Mucho Maas') }
+
+    before do
+      setup_sphinx do
+        [oedipa, mucho].each do |person|
+          ListEntity.create!(list_id: list.id, entity_id: person.id)
+        end
+      end
+    end
+
+    after do
+      teardown_sphinx { delete_entity_tables }
```

Comment:
try running without the "delete_entity_tables"  and see if it still works?

Reply:
It works! Thanks.